.gitignore

```diff
@@ -9,6 +9,7 @@ __pycache__/
 # Large data files
 data/20000_emails.csv
 data/50000_emails.csv
+data/all_emails.csv
```

Comment:
Can you share this on Google Drive and include a link as a PR comment?

Reply:
done